netbox/netbox/views/generic/bulk_views.py

```diff
@@ -170,15 +171,40 @@ def get(self, request):
         # Render the objects table
         table = self.get_table(self.queryset, request, has_bulk_actions)
 
+        # Check for filterset_form(s) on this view and/or the table, if a form exists:
+        # * If both exist, initialize both
+        # * If a filterset form for the table exists, only initialize the table filterset_form
+        # * If a filterset form exists for the view, initialize the filterset form
+        # * Apply to the table for use by the table and initialize a separate instance of the form for use by the table
+        #   column filters
+        # * Otherwise set to None
+        if self.filterset_form and table.filterset_form:
+            filterset_form = self.filterset_form(request.GET)
+            table.filterset_form = table.filterset_form(request.GET)
+        elif self.filterset_form and not table.filterset_form:
+            filterset_form = self.filterset_form(request.GET)
+            table.filterset_form = self.filterset_form(request.GET)
+        elif not self.filterset_form and table.filterset_form:
+            filterset_form = None
+            table.filterset_form = table.filterset_form(request.GET)
+        else:
+            filterset_form = None
+            table.filterset_form = None
```

Comment:
Can we ditch the association of a filterset form on the table directly, and just use the one associated with the view? That would make things a lot cleaner.